web/docker-entry.sh

```diff
@@ -5,5 +5,6 @@ if [ -f "/cert/cert.pem" -a -f "/cert/key-no-password.pem" ]; then
 else
   echo "linking plain config"
 fi
-ln -s /etc/nginx/sites-available/mattermost$ssl /etc/nginx/conf.d/mattermost.conf
+if [ -e "/etc/nginx/conf.d/default.conf" ]; then rm /etc/nginx/conf.d/default.conf; fi
+ln -s /etc/nginx/sites-available/mattermost${ssl} /etc/nginx/conf.d/mattermost.conf
```

Comment:
Good point here, but is that necessary to add this to this PR ?

To keep a correct Git history I will prefer to remove this changes from this Pull Request and to open a new one just for this (or if you don't want, I'll do the commit by myself). I will accept directly ;)